package.json

```diff
@@ -15,6 +15,7 @@
     "localdb:down": "pg_ctl -D apps/passport-server/local-db-data -l apps/passport-server/local-db-log stop",
     "build": "turbo run build",
     "dev": "turbo run dev --parallel",
+    "dev:bot": "concurrently -c auto \"yarn workspace passport-client dev\" \"yarn workspace passport-server dev\"",
```

Comment:
why do we need this command? if we do, let's add some documentation

Reply:
@rrrliu For me it was useful to have logs that were just the client + server. Also I didn't need hot reloading for every package, just passport-client and server. Can't add comments in the JSON, what do you recommend for documentation (if we keep the command)?

Reply:
Gotcha. `dev:bot` is just not super descriptive nor directly clear from the command itself - could we either remove this or rename it the command to `passport:dev`?